### PR TITLE
feat: support MSE Gateway

### PR DESCRIPTION
**Pull request by wangzhuzhen**
#### What this PR does / why we need it:
Erda hepa support MSE Gateway by set dice-cluster-info with config GATEWAY_PROVIDER and PROVIDER_GATEWAY_ENDPOINT.

If set GATEWAY_PROVIDER as "" or not set, Erda use Nginx-Kong as API Gateway as usual.

If set GATEWAY_PROVIDER as "MSE", Erda use Aliyun MSE as API Gateway in Aliyun ACK cluster.

#### Specified Reviewers:

/assign @chengjoey @sfwn @sixther-dc @luobily @iutx 

#### ChangeLog
Feature: Support Aliyun MSE Gateway in Erda hepa （Erda hepa 支持阿里云 MSE Gateway 网关）

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |     Support Aliyun MSE Gateway in Erda hepa     |
| 🇨🇳 中文    |      Erda hepa 支持阿里云 MSE Gateway 网关        |

### CODE REVIEW

File: build/dockerfiles/Dockerfile [hunk 1]
```diff
@@ -43,6 +45,16 @@ RUN \
     npm i -g jackson-converter@1.0.10 && \
     pip3 install dicttoxml xmindparser
 
+RUN if [ "$MODULE_PATH" = "hepa" ]; then \
+      if  [ "$ARCH" = "amd64" ]; then \
+        wget  https://terminus-dice.oss-cn-hangzhou.aliyuncs.com/installer/nginx_1.20.2-1~bullseye_amd64.deb;  \
```
Comment (iutx) [inline build/dockerfiles/Dockerfile:50]: e.g. 
```shell
https://terminus-dice.oss-cn-hangzhou.aliyuncs.com/installer/nginx_1.20.2-1~bullseye_${ARCH}.deb
```

File: internal/tools/orchestrator/hepa/gateway-providers/mse/mse_adapter.go [hunk 2]
```diff
@@ -0,0 +1,484 @@
...
+func NewMseAdapter() gateway_providers.GatewayAdapter {
+	return &MseAdapterImpl{
+		ProviderName: Mse_Provider_Name,
+	}
+}
+
+func (impl *MseAdapterImpl) GatewayProviderExist() bool {
+	//TODO: WZZ
+	// check Deployment mse-ingress-controller/ack-mse-ingress-controller in k8s cluster
+	log.Debugf("GetVersion is not implemented in Mse gateway provider.")
```
Comment (dspo) [inline internal/tools/orchestrator/hepa/gateway-providers/mse/mse_adapter.go:52]: 这个 debug 信息是不是写错了

**Review by dspo — APPROVED** [review]

File: build/dockerfiles/Dockerfile (hunk 1 above)
Comment (wangzhuzhen) [reply build/dockerfiles/Dockerfile:50]: OK

File: internal/tools/orchestrator/hepa/gateway-providers/mse/mse_adapter.go (hunk 2 above)
Comment (wangzhuzhen) [reply internal/tools/orchestrator/hepa/gateway-providers/mse/mse_adapter.go:52]: 嗯，后面调整为对应的方法名称

**Review by sixther-dc — APPROVED** [review]
/approve

**Pull request merged**